docs/pages/component/props.md

```diff
@@ -1,160 +1,179 @@
 
 Platforms: all
 
 ### `audioOutput`
+
 Changes the audio output.
-* **speaker (default)** - plays through speaker
-* **earpiece** - plays through earpiece
+
+- **speaker (default)** - plays through speaker
+- **earpiece** - plays through earpiece
 
 Platforms: Android, iOS, visionOS
 
 ### `automaticallyWaitsToMinimizeStalling`
+
 A Boolean value that indicates whether the player should automatically delay playback in order to minimize stalling. For clients linked against iOS 10.0 and later
-* **false** - Immediately starts playback
-* **true (default)** - Delays playback in order to minimize stalling
+
+- **false** - Immediately starts playback
+- **true (default)** - Delays playback in order to minimize stalling
 
 Platforms: iOS, visionOS
 
 ### `backBufferDurationMs`
+
 The number of milliseconds of buffer to keep before the current position. This allows rewinding without rebuffering within that duration.
 
 Platforms: Android
 
 ### `bufferConfig`
+
 Adjust the buffer settings. This prop takes an object with one or more of the properties listed below.
 
-Property | Type | Description
---- | --- | ---
-minBufferMs | number | The default minimum duration of media that the player will attempt to ensure is buffered at all times, in milliseconds.
-maxBufferMs | number | The default maximum duration of media that the player will attempt to buffer, in milliseconds.
-bufferForPlaybackMs | number | The default duration of media that must be buffered for playback to start or resume following a user action such as a seek, in milliseconds.
-bufferForPlaybackAfterRebufferMs | number | The default duration of media that must be buffered for playback to resume after a rebuffer, in milliseconds. A rebuffer is defined to be caused by buffer depletion rather than a user action.
-maxHeapAllocationPercent | number | The percentage of available heap that the video can use to buffer, between 0 and 1
-minBackBufferMemoryReservePercent | number | The percentage of available app memory at which during startup the back buffer will be disabled, between 0 and 1
-minBufferMemoryReservePercent | number | The percentage of available app memory to keep in reserve that prevents buffer from using it, between 0 and 1
+| Property                          | Type   | Description                                                                                                                                                                                     |
+| --------------------------------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| bufferSize                        | number | The maximum cache size for Android to use as a cache folder, in MB.                                                                                                                             |
```

Comment:
Can you be a little bit more precise please ?
Maybe something like:

Cache size in MB, it will allow applications to store video data for a while in the cache folder, it is useful to decrease bandwidth usage when repeating small videos. **Android only**.

I am not sure of the naming also...
Can you confirm this is a cache, not a buffer? My issue is that next props deals with buffer (which is stored in ram) where here we store in cache folder on the phone...

Maybe cacheSizeMB would be a better naming